03-task-manager/starter/controllers/ejemploController.js

```diff
@@ -47,20 +34,15 @@ exports.updateFullName = async (req, res) => {
     const updatedExample = await Example.findByIdAndUpdate(_id, { nombre: nombre }, { new: true });
 
     if (!updatedExample) {
-      return res.status(404).json({ message: 'The User was not found.' });
+      return next(createCustomError(`No name with Id: ${nombreID} was found..`, 404));
     }
 
     res.json(updatedExample);
-  } catch (error) {
-    console.log(error);
-    res.status(500).send(error);
-  }
-};
+});
 
 // Controller to delete an example by ID
-exports.deleteFullName = async (req, res) => {
-  try {
-    const { _id } = req.params;
+exports.deleteFullName = asyncWrapper (async (req, res) => {
```

Comment:
Same thing here, this route doesn't work.  The error is that _id is not defined.  You should be finding and deleting the entry with the request paramater before checking for the existence of your deletedExample.  You also need to be passing 'next' in as a parameter.  Please, please test your routes before submitting.